src/StudyRegistration/StudyRegistrationRequestForm.tsx

```diff
@@ -294,6 +296,11 @@ const StudyRegistrationRequestForm: React.FunctionComponent<StudyRegistrationPro
               </Button>
             </Space>
           </Form.Item>
+          { (studyRegistrationConfig.studyRegistrationFormDisclaimerField) &&
+              <Typography style={{ textAlign: 'center' }}>
```

Comment:
I think for this disclaimer texts we should give them a smaller font size so they can be distinguished from other texts on this form. `12px` or `0.8rem` should be a good choice

Apply styles to this component using the CSS stylesheet so it can be override if needed